paper-api/src/main/java/io/papermc/paper/registry/data/InstrumentRegistryEntry.java

```diff
@@ -0,0 +1,90 @@
+package io.papermc.paper.registry.data;
+
+import io.papermc.paper.registry.RegistryBuilder;
+import net.kyori.adventure.sound.Sound;
+import net.kyori.adventure.text.Component;
+import org.bukkit.MusicInstrument;
+import org.checkerframework.checker.index.qual.NonNegative;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Contract;
+
+/**
+ * A data-centric version-specific registry entry for the {@link org.bukkit.MusicInstrument} type.
+ */
+@ApiStatus.Experimental
+@ApiStatus.NonExtendable
+public interface InstrumentRegistryEntry {
+
+    @Contract(pure = true)
+    @NonNegative
+    float duration();
+
+    @Contract(pure = true)
+    @NonNegative
+    float range();
+
+    Component description();
+
+    Sound sound();
```

Comment:
This is incorrect. We cannot/should not use the Sound type here. This has to be an Either<SoundEventRegistryEntry, TypedKey<Sound>>. See how I've implemented it in JukeboxSoundRegistryEntry. https://github.com/PaperMC/Paper/pull/11805/files#diff-806c67f8d37d5fadd51ad314002172c70d3eff1ce45f2fe2c7219f27b9ad9b07R30

Reply:
ohhh i see... first time dealing with this. but then this PR now depends in https://github.com/PaperMC/Paper/pull/11805 for the SoundEventRegistryEntry not?